zuul.d/projects.yaml

```diff
@@ -12,7 +12,6 @@
         - edpm-ansible-molecule-edpm_module_load
         - edpm-ansible-molecule-edpm_kernel
         - edpm-ansible-molecule-edpm_libvirt
-        - edpm-ansible-molecule-edpm_nova
```

Comment:
this is very high risk.
disabling the job should be avoided.

you can disable the failing test or make it non voting but this should effectively never be the approch.

Reply:
I agree, however this job is no more functional anyway, unless fixed.

But I updated this with a fix instead of dropping the test
https://github.com/openstack-k8s-operators/edpm-ansible/pull/896/commits/aa702f82e632551f175c85186a8ee2c851d86d09